[Feature #157694761] Set Up Help Center With Frequently Asked Questions

daktari01 opened the pull request:
### What does this PR do?
Adds a help center for the user with Frequently Asked Questions.

### Description of Task to be completed?
The user requires a help center with Frequently Asked Questions, so as to be able to get around the system and learn how to use it quickly.

### How should this be manually tested?
Once the server is running, click on the `http://localhost:8000` link. Log in and click the FAQ tab. Frequently Asked Questions are listed as accordions with the accordion bodies as responses to the questions. 

### What are the relevant pivotal tracker stories?
- [User Help Center](https://www.pivotaltracker.com/story/show/157694761)

### Screenshots
- FAQ Page
<img width="1022" alt="screen shot 2018-06-14 at 08 24 35" src="https://user-images.githubusercontent.com/29580769/41393201-4e2ec750-6fad-11e8-98f8-c3b38f53f7db.png">

@Awesome94 @Felistas @sam-karis @kzyGit @tonykimathi @muhozi

daktari01 committed:
[feature #157694761] Set up accordions with titles
The user clicks on a question
Then the response is displayed as a collapse

daktari01 committed:
[feature #157694761]Add FAQ page
Add frequently asked questions as accordion titles
Add responses as the accordion body.
When the FAQ tab is clicked to open the Faq page,
the first accrdion opens up by default

david-kironde-deel commented on hc/api/migrations/0030_merge_20180613_1942.py:
```diff
@@ -0,0 +1,16 @@
+
+from django.db import migrations
+
+
```
remove double line spacing

david-kironde-deel commented on hc/api/migrations/0029_merge_20180608_1209.py:
```diff
@@ -0,0 +1,16 @@
+
+from django.db import migrations
+
+
```
remove double line spacing

david-kironde-deel commented on hc/api/migrations/0029_auto_20180613_1446.py:
```diff
@@ -0,0 +1,20 @@
+
+from django.db import migrations, models
+
+
```
remove double line spacing

daktari01 committed:
[chore #157694761] Add hyperlinks
Use hyperlinks to improve user experience
When I mention 'Home' I give a link to it
So that a user does not have to look for the link
throughout the system
